File: tys-application/tys-commons/src/main/java/co/com/mintrabajo/tys/commons/domain/DatosFormacionAdolecente.java
package co.com.mintrabajo.tys.commons.domain;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;

import java.util.Date;

@Data
@AllArgsConstructor
@Builder(builderMethodName = "newInstance")
public class DatosFormacionAdolecente {
    private String nombreInstitucion;
    private String nit;
    private Long idTipoEducacion;
    private Long idTipoInstitucion;
    private String numeroTelefonico;
    private Direccion direccion;
    private String nombreProgramaAcademico;
    private String grado;
    private Long idPeriodoAcademico;
    private String anio;
    private String fechaInicio;
    private String fechaFin;
}

File: tys-spa/ciudadano/src/app/domain/datosFormacionAdolecente.ts
import {Direccion} from './direccion';

export interface DatosFormacionAdolecente {
  nombreInstitucion: string;
  nit: string;
  idTipoEducacion: number;
  idTipoInstitucion: number;
  numeroTelefonico: string;
  direccion: Direccion;
  nombreProgramaAcademico: string;
  grado: string;
  idPeriodoAcademico: number;
  anio: string;
  fechaInicio: any;
  fechaFin: any;
}
